controllers/testdata/logs.json

```diff
@@ -172,16 +177,9 @@
                 "readOnly": true
               },
               {
-                "name": "cdap-cm-vol-my-config-map-1",
```

Comment:
nit: is this a cleanup ? Cannot tell why it is related to your change

Reply:
This is unrelated, but the init containers of the logging service should not have this volume mount. This likely slipped through (and still passes) because we are checking for the expected JSON template to be a superset of the output JSON, when in fact we should be checking the opposite for our integration tests. I've gone ahead and opened [CDAP-18732](https://cdap.atlassian.net/browse/CDAP-18732) to clean this up in the future.